common/libs/values.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import type Big from 'big.js';
+
+export function padLeft(n: string, width: number, z: string = '0'): string {
+  return n.length >= width ? n : new Array(width - n.length + 1).join(z) + n;
+}
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
bignumber.js has a toString that can make it hex. This is what v3 uses: https://github.com/MikeMcl/bignumber.js/